### PR TITLE
Removed toggle button from dimmer widget

### PR DESCRIPTION
**Pull request by aounhaider1**
fixes https://github.com/eclipse/smarthome/issues/1808
Signed-off-by: Aoun Bukhari <bukhari@itemis.de>

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html [hunk 1]
```diff
@@ -56,14 +56,6 @@ <h3 class="clabel">{{thing.label}}</h3>
 													</div>
 												</div>
 												<md-slider class="md-default-theme" style="height: 40px;" flex min="0" max="100" ng-model="item.state" ng-change="setBrightness(item.state)" aria-label={{item.label}}></md-slider>
-												<div layout="row" layout-align="start center" class="dimmer-widget" ng-if="!isHideSwitch(item.category)">
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html]: I only noticed now that there is already a feature "isHideSwitch" implemented, which allows hiding the toggle based on the category - imho it makes sense to keep this feature in place.

I would only suggest to change the default: If there is no exception defined in the category list, the toggle should be hidden. Only if a "showSwitch=true" is defined, it should be shown. For now, I would show it for DimmableLights only.

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js [hunk 2]
```diff
@@ -222,7 +223,7 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
         'SoundVolume' : {
             label : 'Volume',
             icon : 'volume_up',
-            hideSwitch : true
+            showSwitch : false
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: As "false" is now our default, we can remove this line here, right?

Comment (aounhaider1) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: Yes, if we remove this line the toggle will not be displayed in the same way as if we set it to false.

Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: ok, then please remove it as it is superfluous.

Comment (aounhaider1) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: Removed.

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**